ferretdb/handler_pg.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ferretdb
+
+import (
+	"time"
+
+	"github.com/FerretDB/FerretDB/internal/handlers"
+	"github.com/FerretDB/FerretDB/internal/handlers/pg"
+	"github.com/FerretDB/FerretDB/internal/handlers/pg/pgdb"
+)
+
+// init registers `pg` handler for PostgreSQL that is always enabled.
+func init() {
```

Comment:
Let's move handlers registration to shared internal package

Reply:
> Let's move handlers registration to shared internal package

How do you see it? 
Functions in the package + init-s that call the functions?

Reply:
> Yes, plus blank-import for side-effects. Just like with database/sql drivers, for example

works w/o blank imports.